Project installation fails with jdk21

Installation fails with this version:
```
java 21.0.5 2024-10-15 LTS
Java(TM) SE Runtime Environment (build 21.0.5+9-LTS-239)
Java HotSpot(TM) 64-Bit Server VM (build 21.0.5+9-LTS-239, mixed mode, sharing)
```

Error:
```
INFO] Reactor Summary for Backend Core 1.1.0-SNAPSHOT:
[INFO]
[INFO] Backend Core ....................................... SUCCESS [  0.438 s]
[INFO] Backend Core - Model ............................... FAILURE [  0.467 s]
[INFO] Backend Core - Data Access Interfaces .............. SKIPPED
[INFO] Backend Core - Data Access Layer Implementations ... SKIPPED
[INFO] Backend Core - Business Interfaces ................. SKIPPED
[INFO] Backend Core - Business Implementations ............ SKIPPED
[INFO] Backend Core - Business Implementation (Spring Data) SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.955 s
[INFO] Finished at: 2025-09-19T10:19:27-03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project backend-core-model: Fatal error compiling: java.lang.NoSuchFieldError: Class com.sun.tools.javac.tree.JCTree$JCImport does not have member field 'com.sun.tools.javac.tree.JCTree qualid' -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :backend-core-model
```

--- a/pom.xml
+++ b/pom.xml
@@ -72,7 +72,7 @@
 			<dependency>
 				<groupId>org.projectlombok</groupId>
 				<artifactId>lombok</artifactId>
-				<version>1.18.24</version>
+				<version>1.18.32</version>
 			</dependency>
 		</dependencies>
 	</dependencyManagement>
@@ -102,7 +102,7 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-compiler-plugin</artifactId>
-					<version>3.7.0</version>
+					<version>3.13.0</version>
 				</plugin>
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>

